core/src/main/java/com/yahoo/imapnio/async/request/ImapRequest.java

```diff
@@ -3,8 +3,7 @@
 import java.util.concurrent.ConcurrentLinkedQueue;
 
 import javax.annotation.Nonnull;
-
-import org.apache.avro.reflect.Nullable;
+import javax.annotation.Nullable;
```

Comment:
Also remove the maven dependency from `pom.xml` since it's no longer needed

Reply:
I don't think pom.xml contains avro since it is common used library.